### PR TITLE
contracts: improve events for the client

### PR DESCRIPTION
**Pull request by smartcontracts**
## Description
First PR of several that will improve various events for simplicity on the client side.

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

**Comment by kfichter** [comment]
I couldn't resist doing a little cleanup while I was at it. Sorry!

File: contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol [hunk 1]
```diff
@@ -348,7 +348,10 @@ contract OVM_StateCommitmentChain is iOVM_StateCommitmentChain, Lib_AddressResol
             batchHeader.batchRoot,
             batchHeader.batchSize,
             batchHeader.prevTotalElements,
-            batchHeader.extraData
+            batchHeader.extraData,
```
Comment (tynes) [inline contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol:351]: Extradata is dead code iirc

**Comment by tynes** [comment]
CI is failing, this is something weird:

```
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 14000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 3000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 4000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 5000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 6000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 7000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 8000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 9000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 10000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 11000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
batch_submitter_1    | (node:1) TimeoutOverflowWarning: 12000000000 does not fit into a 32-bit signed integer.
batch_submitter_1    | Timeout duration was set to 1.
```

**Review by ben-chain — COMMENTED** [review]
LGTM other than CI issues!

File: contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol (hunk 1 above)
Comment (tynes) [reply contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol:351]: We should delete the `extraData` if its not being used

**Comment by smartcontracts** [comment]
Integration tests fail because the event changes :-(

**Pull request closed without merging**